script.js

```diff
@@ -0,0 +1,66 @@
+const bookAdd = document.querySelector('.book');
+const title = document.querySelector('#title');
+const author = document.querySelector('#author');
+const btn = document.querySelector('.submit');
+
+const collection = [];
+
+/* eslint-disable no-use-before-define */
+
+// Add book
+const updateUI = () => {
+  bookAdd.innerHTML = '';
+  collection.forEach((info, index) => {
+    const newBook = document.createElement('div');
+    newBook.classList.add('new-book');
+    const para = document.createElement('p');
+    para.textContent = `${info.title} By ${info.author}`;
+    const button = document.createElement('button');
+    button.textContent = 'Remove';
+    newBook.appendChild(para);
+    newBook.appendChild(button);
+    bookAdd.appendChild(newBook);
+    button.addEventListener('click', removeBook.bind(index));
+  });
+};
+
+btn.addEventListener('click', (event) => {
+  event.preventDefault();
+  if (title.value && author.value) {
+    const obj = { title: title.value, author: author.value };
+    collection.push(obj);
+    updateUI(collection);
+    title.value = '';
+    author.value = '';
+  }
+});
+
+// Remove book
+const removeBook = function () {
+  collection.splice(this, 1);
+  updateUI(collection);
+};
+
+// Local storage
+const titleInput = document.getElementById('title');
+const authorInput = document.getElementById('author');
+
+function handleChange() {
+  const formData = {
+    titleData: titleInput.value,
+    authorData: authorInput.value,
+  };
+  localStorage.setItem('form', JSON.stringify(formData));
+}
+
+document.addEventListener('DOMContentLoaded', () => {
+  const getFormValue = localStorage.getItem('form');
+  if (getFormValue) {
+    const formObject = JSON.parse(getFormValue);
+    titleInput.value = formObject.titleData;
+    authorInput.value = formObject.authorData;
+  }
+});
```

Comment:
- [ ] You are yet to implement the local storage system as requested. When I add a title and author, it is saved to the local storage. However, when I refresh the page, instead of rendering a book, the title and author values are inserted into the inputs. Kindly fix this behavior.

  Book is added | Page is refreshed
  :---:|:---:
  ![swe-1](https://user-images.githubusercontent.com/54848363/176014223-59019d4b-ed02-49a5-aaf5-d21d73bae8fe.jpg) | ![swe-2](https://user-images.githubusercontent.com/54848363/176014222-7ef4488b-4adb-4470-a108-59bbb1c72e46.jpg)

- [ ] When a book is deleted it does not delete from the local storage as well, and so when the page is refreshed, it behaves as highlighted above. Kindly fix that.

- [ ] When the first book has been added and I add another one, the local storage only stores the last book. In other words the storage saves ONLY ONE book at a time. Kindly fix that.

  ![swe-3](https://user-images.githubusercontent.com/54848363/176014216-170aad95-315d-4e4a-a17c-0a25800e6d4e.jpg)